app/views/layouts/application.html.erb

```diff
@@ -3,6 +3,7 @@
 <head>
   <title>RailsStaffGroupProject</title>
   <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <script src="https://kit.fontawesome.com/5ca745b1f9.js" crossorigin="anonymous"></script>
```

Comment:
Alternatively, you can look into the Font Awesome Rails gem, which are usually better than scripts

https://github.com/bokmann/font-awesome-rails

Reply:
Removed; not currently using icons